src/main/java/pl/tomaszdziurko/jvm_bloggers/settings/SettingKeys.java

```diff
@@ -2,7 +2,7 @@
 
 public enum SettingKeys {
 
```

Comment:
This change brings nothing except broken formatting (we are using 4 spaces indentation instead 3). Please revert to original.